src/react-components/input/Button.stories.js

```diff
@@ -44,3 +47,17 @@ export const Tokens = () => (
     </Button>
   </Column>
 );
+
+export const Landing = () => (
+  <Column padding>
+    <Button preset="signin" thick>
+      sign in/sign up
```

Comment:
Why not capitalized?

Reply:
https://www.figma.com/file/J6YKkb2vWDlk81zm4kxUzA/Hubs-discoverability-and-onboarding?node-id=375%3A0

Reply:
Ah. I would push back on that choice because all the other buttons are capitalized.

Reply:
Also this should be a `<FormattedMessage />` component -- it should be throwing a linter error. If it's not throwing an error, lmk.